graphql.go

```diff
@@ -275,6 +283,9 @@ func (c *Client) runWithJSON(ctx context.Context, req *Request, resp interface{}
 	}
 	r.Header.Set("Content-Type", "application/json; charset=utf-8")
 	r.Header.Set("Accept", "application/json; charset=utf-8")
+	for key, value := range c.defaultHeader {
+		r.Header.Add(key, value)
```

Comment:
is the second argument in `Add()` an `interface{}` type? If so, we should make the defaultHeaders map of type `map[string]interface{}` for ease of use

Reply:
The 2nd argument is a string type.